angular-single-page-app/Dockerfile

```diff
@@ -1,19 +1,3 @@
-# Copyright (c) 2023, WSO2 LLC. (https://www.wso2.com/) All Rights Reserved.
```

Comment:
Any reason for removing the licence header?

Reply:
Sorry. It was a mistake